src/ontology/mondo-edit.obo

```diff
@@ -533518,6 +533520,23 @@ is_a: MONDO:0005583 {source="OMIA:002138"} ! non-human animal disease
 intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
 intersection_of: MONDO:0700097 MONDO:0005620 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog cerebral amyloid angiopathy
 
+[Term]
+id: MONDO:1010029
+name: aleutian mink disease, human
+def: "A human disease caused by infection with Aleutian mink disease parvovirus." [MONDO:patterns/infectious_disease_by_agent, PMID:19961696]
+comment: In Mondo we make the distinction between human and non-human animal diseases. Cross-references do not necessarily make this distinction. Annotate to the human or non-human animal term as appropriate. This disease is considered zoonotic. {source="PMID:19961696"}
```

Comment:
'comment' should not have a source. Instead, please add the PMID in the text.
Here is an alternate text:
This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:19961696). In Mondo, we distinguish between human and non-human animal diseases, and we currently do have a general term to represent a species-agnostic version of the disease. This record is for the human version of “aleutian milk disease”.

Reply:
Changed milk to mink and revised to: 
This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:19961696). In Mondo, we distinguish between human and non-human animal diseases, and we currently do not have a general term to represent a species-agnostic version of the disease. This record is for the human version of “aleutian mink disease”.